src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -867,6 +867,92 @@ internal sealed partial class LazyBinaryOperatorExpression : BaseBinaryOperatorE
         protected override IOperation RightOperandImpl => _lazyRightOperand.Value;
     }
 
+    /// <summary>
+    /// Represents an operation with two operands that produces a result with the same type as at least one of the operands.
+    /// </summary>
+    internal abstract class BaseTupleBinaryOperatorExpression : Operation, ITupleBinaryOperation
+    {
+        public BaseTupleBinaryOperatorExpression(BinaryOperatorKind operatorKind, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit)
+            : base(OperationKind.BinaryOperator, semanticModel, syntax, type, constantValue, isImplicit)
+        {
+            OperatorKind = operatorKind;
+        }
+        /// <summary>
+        /// Kind of binary operation.
+        /// </summary>
+        public BinaryOperatorKind OperatorKind { get; }
+        /// <summary>
+        /// Left operand.
+        /// </summary>
+        public IOperation LeftOperand => Operation.SetParentOperation(LeftOperandImpl, this);
+        /// <summary>
+        /// Right operand.
+        /// </summary>
+        public IOperation RightOperand => Operation.SetParentOperation(RightOperandImpl, this);
+
+        protected abstract IOperation LeftOperandImpl { get; }
+        protected abstract IOperation RightOperandImpl { get; }
+
+        public override IEnumerable<IOperation> Children
```

Comment:
>Children [](start = 48, length = 8)

Do we have explicit test for this API?

Reply:
I did, but I can't find it :-(
I must have messed up a git operation...

---
In reply to: [173929906](https://github.com/dotnet/roslyn/pull/25374#discussion_r173929906) [](ancestors = 173929906)